packages/squiggle-lang/src/fr/common.ts

```diff
@@ -85,4 +86,30 @@ myFn = typeOf({|e| e})`,
       ),
     ],
   }),
+  maker.make({
+    name: "try",
+    description:
+      "Try to run a function and return its result. If the function throws an error, return the result of the fallback function instead.",
+    definitions: [
+      makeDefinition(
+        [
+          frNamed("fn", frLambdaTyped([], frAny({ genericName: "A" }))),
+          // in the future, this function could be called with the error message
+          frNamed("fallbackFn", frLambdaTyped([], frAny({ genericName: "A" }))),
```

Comment:
Not sure that we should use a generic like this here.

As a user, I'd probably be inclined to use a different type in a lot of cases.

Like, ``try({|| 2+''}, {|| {type: error, message: "X failed" })``

Reply:
Yeah, technically the output type should be a union of both functions' return types, but we don't have unions in FR.

Reply:
What about ``frOr()``?

Reply:
Fixed here: https://github.com/quantified-uncertainty/squiggle/commit/22f74357f0f2e86f5c320875a36356225a851e94